### PR TITLE
The vote cannot be modified if ballots are issued #160

### PR DESCRIPTION
**Pull request by valentinbujdoso**

### CODE REVIEW

File: src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java [hunk 1]
```diff
@@ -48,5 +48,20 @@ public void invalid_voteId_is_rejected() throws ReportedException {
 					invalidvoteId, adminInfo.adminKey, voteName)
 		).assertMessageIs(String.format("illegal voteId: %s",invalidvoteId));
 	}
+	
+	@tested_feature("Manage votes")
+	@tested_operation("modify vote")
+	@tested_behaviour("The vote cannot be modified if there are ballots issued.")
+	@Test
+	public void proper_voteId_adminKey_and_voteName_with_ballot_does_not_modify_vote() throws ReportedException {
```
Comment (magwas) [inline src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java]: the test check for an exception thrown
the description says the vote will not be modified. it is not checked, and it have multiple aspects:
1. ballot is kept
2. cast vote is not recorded.

**Review by magwas — CHANGES_REQUESTED** [review]

File: src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java (hunk 1 above)
Comment (valentinbujdoso) [reply src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java]: So I have to extend this testcase with 2 more checks?

Comment (magwas) [reply src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java]: You should rename this test, and yes, add two tests for the other behaviours.

Comment (valentinbujdoso) [reply src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java]: I fixed the test name, but I dont get your aspects.
Could you explain a little bit more detailed?

Comment (valentinbujdoso) [reply src/test/java/org/rulez/demokracia/pdengine/VoteModificationValidationTest.java]: I mean I just did the #160 issue. I don't see anything about cast vote and ballot kept theme.

**Review by magwas — APPROVED** [review]

**Comment by magwas** [comment]
testcaseCount=1

**Pull request merged**